src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CompilerExtensions.projitems

```diff
@@ -489,6 +489,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)Utilities\Contract.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Utilities\Contract.InterpolatedStringHandlers.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Utilities\ISpeculationAnalyzer.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)utilities\SingleInitNullable.cs" />
```

Comment:
@tmeschter i've been getting this issue rarely, but persistently.  Files get added to the projitems, but part of the path is lowercased.  i can't figure it out at all.  highly annoying.  does some part of CPS lowercase paths?  It breaks things on our unix builds.

Reply:
Note that this is a shared project and not a normal csproj.